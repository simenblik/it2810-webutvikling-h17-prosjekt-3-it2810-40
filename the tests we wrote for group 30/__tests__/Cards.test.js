import React from 'react';
import { shallow, mount, render } from 'enzyme';
import toJson from 'enzyme-to-json';
import Cards from '../src/Components/Cards.js';

/* Takes a snapshot of the rendered component and see if it
matches the original snapshot.
*/
describe('<Cards/>, functions', () => {
	it('should match to previous snapshots', () => {
		const component = shallow(<Cards />);
		const tree = toJson(component);
		expect(tree).toMatchSnapshot();
	});
});
