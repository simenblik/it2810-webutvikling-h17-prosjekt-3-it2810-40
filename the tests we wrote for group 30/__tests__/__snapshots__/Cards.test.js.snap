// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cards/>, functions should match to previous snapshots 1`] = `
<div
  className="cardContainer"
>
  <a
    href="/Todo"
  >
    <h2>
      Todo list!
    </h2>
    <img
      alt="todo"
      id="todoCard"
      src="https://openclipart.org/image/2400px/svg_to_png/28096/sheikh-tuhin-To-Do-List.png"
    />
  </a>
  <a
    href="/Appointments"
  >
    <h2>
      Appointments!
    </h2>
    <img
      alt="appointment"
      id="appointmentCard"
      src="http://www.calcuttayellowpages.com/cimage26/109881appoint.png"
    />
  </a>
  <a
    href="/Notes"
  >
    <h2>
      Notes!
    </h2>
    <img
      alt="notes"
      id="noteCard"
      src="https://upload.wikimedia.org/wikipedia/commons/9/9d/Notepad-117597_640.png"
    />
  </a>
</div>
`;
